pcdsdevices/pneumatic.py

```diff
@@ -0,0 +1,69 @@
+
+"""
+Pneumatic Classes.
+
+This Module contains all the classes relating to Pneumatic Actuators
+"""
+
+from lightpath import LightpathState
+from ophyd import Component as Cpt
+from ophyd import EpicsSignal, EpicsSignalRO
+
+from pcdsdevices.interface import LightpathMixin
+
+
+class BeckhoffPneumatic(LightpathMixin):
```

Comment:
Adds some standard utilities like the terminal pretty-print
```suggestion
class BeckhoffPneumatic(BaseInterface, LightpathMixin):
```

Reply:
done.